portfolio/bin/src/main/webapp/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Lilian Hunt</title>
```

Comment:
Still the 4-space problem throughout this file.

Reply:
fixed